freqtrade/exchange/exchange.py

```diff
@@ -269,6 +269,15 @@ def validate_timeframes(self, timeframe: List[str]) -> None:
         """
         Checks if ticker interval from config is a supported timeframe on the exchange
         """
+        if not hasattr(self._api, "timeframes") or self._api.timeframes is None:
```

Comment:
Technically looks good - but you'll want to remove lines 281-286 (they do the same thing, but worded differently).

I assume that was your intent but during testing you added it and then forgot to remove ...

Reply:
Ah, shame on mee, thanks! 

I forgotten it (to remove the original part of the code) when applied the change manually from the stash bunch of changes printed as a patch, stashed with other changes from some other local working branch, where I made this change first...